java/settings.gradle

```diff
@@ -8,3 +8,4 @@
  */
 
 rootProject.name = 'uacs_incentives'
+includeBuild("../../math")
```

Comment:
Can you get rid of this, please?

Reply:
Done

Reply:
in bdfc845de00631b27ca6c12e1c5e2154d65a2d92